### ISSUE TITLE
Can you support image compression?

### ISSUE THREAD

**Issue by mxz94** [Thinkmill/keystatic#1059]
Can you support image compression?

**Comment by jossmac** [comment]
Yes, via [cloud images](https://keystatic.com/docs/cloud#cloud-images)!

Image storage, optimisation, and delivery is one of the features of Keystatic Cloud—a product that integrates with Keystatic, offering additional functionality.

**Comment by mxz94** [comment]
I just want to save the image compressed with js to github

**Comment by jossmac** [comment]
Could you explain in more detail what that might look like? Include screenshots, code snippets, links to existing implementations; anything that will help convey your assumptions and expectations.

**Comment by mxz94** [comment]
This is my keystatic.config.ts configuration
![image](https://github.com/Thinkmill/keystatic/assets/127198845/ab12e0ce-8838-438f-b8f4-de6e13e7f9b9)

I wrote an article and uploaded a 1.32MB image.
![image](https://github.com/Thinkmill/keystatic/assets/127198845/7c1dae1a-5ac6-48e7-8dd6-12f80856964e)

Then I have the original file in my github directory, and I'd like this image to be compressed a little smaller (you can set the ratio)
![image](https://github.com/Thinkmill/keystatic/assets/127198845/8decb75c-3fb6-4f79-b8c0-13cd43b12cb0)
